src/librustc_codegen_ssa/back/link.rs

```diff
@@ -353,6 +352,10 @@ fn link_rlib<'a, B: ArchiveBuilder<'a>>(sess: &'a Session,
         RlibFlavor::Normal => {
             // Instead of putting the metadata in an object file section, rlibs
             // contain the metadata in a separate file.
+            eprintln!("link_rlib -> emit_metadata");
+            // njn: do I need to issue a directive for this metadata? I don't
+            // think so, because this file is going to end up in the archive
+            // file...
```

Comment:
Right yeah this is just a temporary file which goes into an rlib, so no need for a directive here